Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -313,6 +315,41 @@ function vtkResliceCursorWidget(publicAPI, model) {
     publicAPI.modified();
   };
 
+  /**
+   * Allow to rotate a specified plane around an other specified plane
+   * @param {String} planeToRotate Define which plane will be rotated ('X', 'Y', 'Z')
+   * @param {Number} rotateAngle Applied angle in radian
+   * @param {String} rotatedAxisName Define around which normal plane the 'planeToRotate' will be rotated ('X', 'Y', 'Z')
+   */
+  publicAPI.rotatePlane = (planeToRotate, rotateAngle, rotatedAxisName) => {
+    if (planeToRotate === rotatedAxisName) {
+      return;
+    }
+    const normal = model.widgetState[`get${planeToRotate}PlaneNormal`]();
+    const rotatedAxis = model.widgetState[`get${rotatedAxisName}PlaneNormal`]();
+    const newNormal = rotateVector(normal, rotatedAxis, rotateAngle);
+    model.widgetState[`set${planeToRotate}PlaneNormal`](newNormal);
+    if (model.widgetState.getKeepOrthogonality()) {
+      const associatedplaneToRotate = getAssociatedPlaneName(
+        planeToRotate,
+        rotatedAxisName
+      );
+      const associatedNormal = model.widgetState[
+        `get${associatedplaneToRotate}PlaneNormal`
+      ]();
+      const newAssociatedNormal = rotateVector(
+        associatedNormal,
+        rotatedAxis,
+        rotateAngle
+      );
+      model.widgetState[`set${associatedplaneToRotate}PlaneNormal`](
+        newAssociatedNormal
+      );
+    }
+    updateState(model.widgetState);
+    publicAPI.modified();
```

Comment:
It is not clear to me when modified should be called after calling `updateState`. Sometimes it is called, sometimes not. What's the rule ?

Reply:
To be honest, I thought it was to update the rendering. But it seems to work without. Especially if you update the other views...